doc/Programming-guide/Key-platform-features/Notifications/index.md

```diff
@@ -52,66 +63,48 @@ The default notification schema installed for Kaa applications is empty. For the
 }
 ```
 
-You can configure your own notification schema via [REST API]({{root_url}}Programming-guide/Server-REST-APIs/#!/Notifications/saveNotificationSchema) call or use Administration UI as shown below. 
-
-
-The list of notification schemas created by a tenant developer for the application is shown in the **Notification schemas** window.
+To configure your own notification schema, use the [REST API]({{root_url}}Programming-guide/Server-REST-APIs/#!/Notifications/saveNotificationSchema) or select it from the **Notification** page under the **Schemas** section of the application.
```

Comment:
My logic is this: I first say that users can configure the existing schemas and give the screenshot show where they can do it. Then I say that they can create their own schemas, but instead of writing the procedure, I give the link to the similar procedure of creating a clien-side profile schema.